sql/src/test/java/io/crate/analyze/where/WhereClauseAnalyzerTest.java

```diff
@@ -253,248 +232,12 @@ public void testSelectWherePartitionedByColumn() throws Exception {
-        assertThat(whereClause.docKeys().get(), contains(isDocKey(1, 1395874800000L)));
-        // not partitions if docKeys are there
-        assertThat(whereClause.partitions(), empty());
-    }
-
-    @Test
-    public void testSelectFromPartitionedTableWithoutPKInWhereClause() throws Exception {
-        WhereClause whereClause = analyzeSelectWhere("select id from parted where match(name, 'name')");
-        assertThat(whereClause.docKeys().isPresent(), is(false));
-        assertThat(whereClause.partitions(), empty());
-    }
-
-    @Test
-    public void testWherePartitionedByColumn() throws Exception {
-        AnalyzedDeleteStatement statement = e.analyze("delete from parted where date = 1395874800000");
-        WhereClauseAnalyzer whereClauseAnalyzer = new WhereClauseAnalyzer(e.functions(), statement.relation());
-        WhereClause where = whereClauseAnalyzer.analyze(statement.query(), transactionContext);
-        assertThat(where.partitions(),
-            Matchers.contains(new PartitionName("parted", Arrays.asList(new BytesRef("1395874800000"))).asIndexName()));
-    }
-
     @Test
     public void testUpdateWherePartitionedByColumn() throws Exception {
         AnalyzedUpdateStatement update = analyzeUpdate("update parted set id = 2 where date = 1395874800000");
         assertThat(update.query(), isFunction(EqOperator.NAME, isReference("date"), isLiteral(1395874800000L)));
     }
 
-    @Test
-    public void testClusteredByValueContainsComma() throws Exception {
-        WhereClause whereClause = analyzeSelectWhere("select * from bystring where name = 'a,b,c'");
-        assertThat(whereClause.clusteredBy(), contains(isLiteral("a,b,c")));
-        assertThat(whereClause.docKeys().get().size(), is(1));
-        assertThat(whereClause.docKeys().get().getOnlyKey(), isDocKey("a,b,c"));
-    }
-
-    @Test
-    public void testEmptyClusteredByValue() throws Exception {
-        WhereClause whereClause = analyzeSelectWhere("select * from bystring where name = ''");
-        assertThat(whereClause.clusteredBy(), contains(isLiteral("")));
-        assertThat(whereClause.docKeys().get().getOnlyKey(), isDocKey(""));
-    }
-
-    @Test
-    public void testClusteredBy() throws Exception {
-        WhereClause whereClause = analyzeSelectWhere("select name from users where id=1");
-        assertThat(whereClause.clusteredBy(), contains(isLiteral("1")));
-        assertThat(whereClause.docKeys().get().getOnlyKey(), isDocKey("1"));
-
-        whereClause = analyzeSelectWhere("select name from users where id=1 or id=2");
-
-        assertThat(whereClause.docKeys().get().size(), is(2));
-        assertThat(whereClause.docKeys().get(), containsInAnyOrder(isDocKey("1"), isDocKey("2")));
-
-        assertThat(whereClause.clusteredBy(), containsInAnyOrder(isLiteral("1"), isLiteral("2")));
-    }
-
-
-    @Test
-    public void testClusteredByOnly() throws Exception {
```

Comment:
Created `testFilterOnClusteredByColumnDoesNotResultInDocKeysSimpleEq`, `testFilterOnClusteredByColumnDoesNotResultInDocKeysSimpleEqOr` and `testFilterOnClusteredByColumnDoesNotResultInDocKeysIn`